test/http.js

```diff
@@ -403,4 +403,13 @@ describe('assertions', function () {
     }).should.throw('expected cookie \'name2\' to have value \'value\' but got \'value2\'');
 
   });
+
+  it('#charset', function () {
```

Comment:
Can we also add tests where:
- no encoding is specified in the `Content-Type`
- no `Content-Type` is present

Just to test to the `null` case

Reply:
@austince I think I've done what you wanted.